monochart/Chart.yaml

```diff
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.10
+version: 1.2.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: 1.1.7
+appVersion: 1.2.0
```

Comment:
I don't think this should have changed in this context.

Reply:
Sure, I changed it back to 1.1.7. I also changed to `version: 1.1.11` as we can treat addition of missing className as a bugfix. Please review.